### PR TITLE
:mortar_board: Lab9 Headings and Code

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Create lab 9 up to:
- file exists
- added to index
- headings
- relevant code

### Why
Gotta' have a lab 9. 

### Testing
Built local. 

### Additional Notes
Q: _Why are you doing Lab9 in such small bits? Why not do it all at once?_
A: Getting what you can done while having a child not break everything is a win.

### CODE REVIEW

**Comment by twentylemon** [comment]
> Q: _Why are you doing Lab9 in such small bits? Why not do it all at once?_

I'd probably never actually ask that question. Small PRs are generally better.

File: src/site/lab9.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+Binary Node
+===========
+
+
+
+.. code-block:: java
+
+    public static void main(String[] args){
+        BinaryNode<Integer> root = new BinaryNode<>(1);
```
Comment (twentylemon) [inline src/site/lab9.rst:27]: Is the intention here that the tree be actually in order for the in order traversal? left-visit-right gives a jumbling ordering, whereas pre-order will actually be in order here.

Comment (jameshughes89) [reply src/site/lab9.rst:27]: It was not intended, but it's a great idea --- will change.

**Review by twentylemon — APPROVED** [review]
I even drew the tree on my whiteboard. Now when it's wrong, we can both be surprised.

**Comment by jameshughes89** [comment]
> I even drew the tree on my whiteboard. Now when it's wrong, we can both be surprised.
Screenshot from 2022-01-10 17-52-43

**Pull request merged**